src/Views/Form/Themes/Sequoia/sections/form-navigator.php

```diff
@@ -0,0 +1,5 @@
+
+<div class="give-form-navigator">
+	<button class="back-btn"><i class="fas fa-chevron-left"></i></button>
+	<p class="title">Intro</p>
```

Comment:
Can we localize this?

Reply:
For sure! Done, ref: https://github.com/impress-org/givewp/commit/8469f487516f1e0d0e84d1af03d17327fecaffbe